frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -1245,16 +1245,6 @@ def matmul(self, resulting_type: ConversionType, x: Conversion, y: Conversion) -
             }
             self.error(highlights)
 
-        if x.is_encrypted and y.is_encrypted:
-            highlights = {
-                x.origin: "lhs is encrypted",
-                y.origin: (
-                    "rhs is encrypted" if x.origin is not y.origin else "operand is encrypted"
-                ),
-                self.converting: "but encrypted-encrypted matrix multiplications are not supported",
-            }
-            self.error(highlights)
-
```

Comment:
I think you forgot to remove https://github.com/zama-ai/concrete/blob/fcec68ea26ffb5f42a43fc856977aea5cddefbf2/frontends/concrete-python/tests/mlir/test_converter.py#L405-L423

Reply:
thanks! removed